### DISCUSSION TITLE
galaxy brain get

### DISCUSSION THREAD

**Discussion by leejongseok1** [learningssik/pull-request#102] [Q&A]
안녕하세요 포스팅 잘 봤습니다

**Comment by Git-SalahEdd** [comment]
can you mark it as answer please ? for galaxy brain rewards
thanks

**Reply by learningssik** [reply]
#105 #106에 댓글남기시면 채택드릴게요!

**Comment by learningssik** [comment]
감사합니다!! #103 #104에 댓글남기시면 채택드릴게요!
